game/ffmpeg.cc

```diff
@@ -138,14 +138,22 @@ void FFmpeg::operator()() {
 	m_duration = m_formatContext->duration / double(AV_TIME_BASE);
 	audioQueue.setDuration(m_duration);
 	int errors = 0;
+	bool eof = false;
 	while (!m_quit) {
+		if(eof) {
+			// FIXME: use a boost::condition on m_quit
```

Comment:
if m_quit is rewritten as "if(eof) then the comment should be updated as well.

Reply:
eof is meant to stay, we want to keep being here, we just want to be notified in the eof loop when quit is called, we might prefer a boost::condition instead of waking up.

Reply:
the comment here is still wrong tough.

Reply:
No, this comment is correct : we do not want to pull m_quit comparison (with the sleep), but instead use a condition on the quit behavior. It avoids waking up the thread each 100ms

Reply:
@yoda-jm instead of `boost::condition` use `std::condition_variable`

As for the comment, i have a pleasant hate regarding `FIXME` and `TODO` comments since most won't be done. Either fix it with this PR, or create a new issue discussing this (and remove the comment) so this pr can be closed.